ocs_ci/ocs/utils.py

```diff
@@ -624,3 +627,25 @@ def apply_oc_resource(
     log.info(f"Applying rook resource from {template_name}")
     occli = OCP()
     occli.apply(cfg_file)
+
+
+def collect_ocs_logs(dir_name):
+    """
+    Collects OCS logs
+
+    Args:
+        dir_name (str): directory name to store OCS logs. Logs will be stored
+        in dir_name suffix with _ocs_logs.
+
+    Returns:
```

Comment:
You can omit the `Returns` section if it doesn't return anything.

Reply:
Done